reactor/src/Errors.elm

```diff
@@ -57,6 +57,9 @@ viewError error =
         , style "white-space" "pre-wrap"
         , style "background-color" "black"
         , style "padding" "2em"
+        , style "box-sizing" "border-box"
+        , style "width" "100%"
+        , style "overflow" "scroll"
```

Comment:
Should this use `"auto"` so there’s no unnecessary scrollbar when there is no overflow?

The default on macOS is to draw scrollbars on top of content, and only show it when scrolling. By changing a system setting, you can make macOS behave like Windows and Linux, which is to let the scrollbars take space and always be visible.

The difference between `scroll` and `auto` is that if no scrolling is needed, `scroll` shows ugly disabled scrollbars while `auto` shows no scrollbars at all.

Reply:
Good point!